bridge/hardhat.config.js

```diff
@@ -104,8 +123,9 @@ module.exports = {
       url: 'https://public-node.rsk.co',
       blockGasLimit: 6800000,
       gasPrice: 60000000, // 0.06 gwei
-      chainId: 30,
-      hardfork: 'istanbul', // London hardfork is incompatible with RSK gasPrice
+      network_id: chains.RSK_MAIN_NET_CHAIN_ID,
+      token_symbol: 'r',
+      hardfork: 'istanbulchainId.', // London hardfork is incompatible with RSK gasPrice
```

Comment:
Wrong hardfork name, probably copy paste error
```suggestion
      hardfork: 'istanbul.', // London hardfork is incompatible with RSK gasPrice
```

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4